01-introduction.Rmd

```diff
@@ -20,7 +20,7 @@ A simplistic comparison between the different approaches is illustrated in Table
 <!-- Still, many GIS programs ... but most teaching materials focus on the GUI ... to supplement the main graphical user interface (GUI), but  -->
 
 ```{r gdsl, echo=FALSE, message=FALSE}
-d = readr::read_csv("extdata/gis-vs-gds-table.csv")
+d = readr::read_csv("https://github.com/Robinlovelace/geocompr/raw/master/extdata/gis-vs-gds-table.csv")
```

Comment:
@Robinlovelace this is going to break the PDF version.

Reply:
Good point. Happy to revert this. Same with the second point.